stripe/src/main/java/com/stripe/android/model/ShippingInformation.java

```diff
@@ -0,0 +1,78 @@
+package com.stripe.android.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.stripe.android.util.StripeNetworkUtils;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.stripe.android.util.StripeJsonUtils.optString;
+import static com.stripe.android.util.StripeJsonUtils.putStringIfNotNull;
+
+/**
+ * Model representing a shipping address object
+ */
+public class ShippingInformation extends StripeJsonModel {
+
+    private static final String FIELD_ADDRESS = "address";
+    private static final String FIELD_NAME = "name";
+    private static final String FIELD_PHONE = "phone";
+
+    private @Nullable SourceAddress mAddress;
```

Comment:
Yep, the `Card` additions can come later. I'm sorry I didn't catch the `SourceAddress` thing earlier. It feels strange that this type is `SourceAddress` – do you think we can rename in this PR?